features/individual_charity_pages/charity_page.feature

```diff
@@ -61,6 +61,18 @@ Feature: Web page owned by each charity
     Then I should not see "Health" within "what_they_do"
     And I should not see "Education" within "what_they_do"
     And I should not see "Animal Welfare" within "what_they_do"
+    
+  Scenario: display categories as links
+    Then I should see "Health" < tagged > with "a"
+    And  I should see "Education" < tagged > with "a"
+    And  I should see "Voluntary" < tagged > with "a"
+    And  I should see "Finance" < tagged > with "a"
+
```

Comment:
the above scenario is probably redundant if we have the one below, and I wonder if we could reword the below so it didn't rely on the link stucture, i.e. we could click through and expect to see the particular orgs that are in the relevant category ...